pom.xml

```diff
@@ -38,7 +38,7 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<java.version>1.8</java.version>
 		<nexusproxy>https://nexus.edgexfoundry.org</nexusproxy>
-		<repobasepath>content/repositories</repobasepath>
+		<repobasepath>/content/repositories</repobasepath>
```

Comment:
Given that you're using ${repobasepath} prefixed with a / in all later calls, I would suggest not having a leading '/' in there.

Reply:
ah good catch thanks